src/components/Socials/Socials.jsx

```diff
@@ -14,9 +19,12 @@ export default function Socials(){
 	// ---------------------------------
 	const links = data.map(RENDER.item);
 
+	const isVisible = !isSmall || open;
+
 	return (
-		<ul className={ `${s.wrapper} ${open ? s.open : s.closed}` }>
-			{ links }
-		</ul>
+		isVisible &&
+			<ul className={ `${s.wrapper} ${open ? s.open : s.closed}` }>
+				{ links }
+			</ul>
```

Comment:
same as the other component - as it's the only thing being returned then I'd return different stuff rather than doing conditionals inside the return.

OOH, or, here's a point - maybe these conditional renders should be happening in the parent element instead? That way this component doesn't even need to exist until it's rendered...